src/BookReader.tsx

```diff
@@ -74,12 +76,4 @@ export const BookReader: FunctionComponent<BookReaderProps> = (props) => {
     );
 }
 
-const styles = StyleSheet.create({
```

Comment:
You just need to move the declaration of the StyleSheet inside the component

Reply:
Cool! I didn't know you could for some reason. I did it for everything.

Reply:
Oh... actually, conventionally, stylesheets are usually declared after the component, not inside them. (I just offered it because you needed to reference a prop)
Source: https://freecontent.manning.com/applying-and-organizing-styles-in-react-native/

Reply:
Actually, the convention is that the StyleSheet declaration goes after/outside the component.
Source: https://freecontent.manning.com/applying-and-organizing-styles-in-react-native/